### ISSUE TITLE
An update?

### ISSUE THREAD

**Issue by Neustradamus** [TristanCavelier/ghost#1]
Dear @TristanCavelier,

It is possible to update your code to current OS?

Thanks in advance.

**Comment by TristanCavelier** [comment]
Hello @Neustradamus =)
What do you mean by "to the current OS"?
From which OS are you running this script?
Do you have errors or confusing outputs?
Please tell me more ;)

**Comment by Neustradamus** [comment]
@TristanCavelier: Minimum is the current LTS versions:
- https://wiki.ubuntu.com/Releases

Can you look, it is for all ^^

**Comment by TristanCavelier** [comment]
I just checked and I found a old/new version of this script. I just published it.

I checked the latest [LTS Ubuntu (jammy) packages](https://packages.ubuntu.com/jammy/) and the all the dependencies are already installed by default, except for `partclone`.

`awk` is provided by `mawk` package,
`gzip` and `zcat` are provided by `gzip` package,
`lsblk` and `mkswap` by `util-linux`,
`parted` by `parted`,
and the other ones are provided by coreutils or by `sh` itself.

These packages are pretty installed in every OS by default, so other "current OS" must work like Debian, Mint, Kali, ... I'm even pretty sure it also works on Tiny Core Linux.

Feel free to report any other issue ;)
